pom.xml

```diff
@@ -136,7 +136,7 @@
         <dependency>
             <groupId>com.squareup.retrofit2</groupId>
             <artifactId>retrofit</artifactId>
-            <version>2.4.0</version>
+            <version>2.9.0</version>
```

Comment:
Maybe not right now, but we should adopt WebClient (https://docs.spring.io/spring-framework/reference/web/webflux-webclient.html) instead of using retrofit.

Reply:
Yea I saw you using that - hadn't noticed - but I agree